ReactSkia/sdk/CurlNetworking.cpp

```diff
@@ -96,6 +96,25 @@ inline bool CurlRequest::shouldCacheData() {
   return true;
 }
 
+void CurlNetworking::expiryTimerCallback() {
+  double currentTime = getCurrentTime();
+  std::chrono::duration<double, std::milli> milliseconds = Timer::getFutureTime().time_since_epoch();
+  double scheduleTimeExpiry = milliseconds.count();
+  networkCache_->timerEvict(scheduleTimeExpiry,currentTime);
+  if(networkCache_->cacheMapSize()) {
+    scheduleTimeExpiry_ = scheduleTimeExpiry;
+    timer_->reschedule((scheduleTimeExpiry_ - currentTime),0);
+  } else {
+    scheduleTimeExpiry_ = 0;
+  }
+}
+
+double CurlNetworking::getCurrentTime() {
```

Comment:
Can you make as inline function. it is not specific to the curlneworking class as well.